consensus/spos/bls/blsSubroundsFactoryV2.go

```diff
@@ -0,0 +1,65 @@
+package bls
+
+import "github.com/ElrondNetwork/elrond-go/consensus/spos"
+
+// factoryV2 defines the data needed by this factory to create all the subrounds and give them their specific functionality
+type factoryV2 struct {
+	*factory
+}
+
+// NewSubroundsFactoryV2 creates a new factoryV2 object
```

Comment:
why called V2? We will use this also on the regular elrond-go node? If not, let's rename to NewSubroundsFactoryForSovereign

Reply:
This will be used also in the main chain as a new and improved consensus mechanism approach